src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -454,12 +455,10 @@ def get_recovery_slope(template_single, sampling_frequency, peak_idx=None, **kwa
     Return the recovery slope of input waveforms. After repolarization,
     the neuron hyperpolarizes until it peaks. The recovery slope is the
     slope of the action potential after the peak, returning to the baseline
-    in dV/dT. The slope is computed within a user-defined window after
+    in dV/dT. The returned slope is in units of (unit of template)
+    per second. The slope is computed within a user-defined window after
```

Comment:
I actually like below that you put: usually um/s for the distance below. Maybe we put something like if `return_scaled=True` then in uV/s or whatever since that is what we force all voltages to be when scaled? So we give the units for the most common but also make it clear that user's need to pay attention.

Reply:
I've had another go.